lib/open_project/scm/manager.rb

```diff
@@ -46,6 +46,18 @@ def enabled
           registered.select { |scm| Setting.enabled_scm.include?(scm) }
         end
 
+        ##
+        # Return a hash of all managed paths for SCM vendors
+        # { Vendor: <Path> }
+        def managed_paths
+          paths = {}
+          @scms.each do |vendor, klass|
+            paths[vendor] = klass.managed_root if klass.manageable?
```

Comment:
For the use-case I employ, I argue it would be useful to show all registered Scms that are configured to be manageable ,as this information is only shown to an administrator (who may control what SCMs are enabled).